CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Changelog
 
+## Version 0.29.0
+
+### Summary
+
+#### Update Pauli Twirling
+Thanks to @purva-thakre for updating Mitiq's PT functions, clarifying that PT is a noise tailoring technique and for consolidating utilities to be shared between PT and other techniques.
```

Comment:
```suggestion
Thanks to @purva-thakre for updating Mitiq's PT functions, clarifying that PT is a noise tailoring technique and for consolidating utilities to be shared between PT and other techniques.
This release removes the `execute_with_pt` functionality in replace of `pauli_twirl_circuit`.
```

Reply:
Agree, good to be precise about what changed. Added in latest commit.